.prettierrc.js

```diff
@@ -0,0 +1 @@
+
```

Comment:
Here's the prettier config from odin-components...if you want to look at prettier and see other options (especially for `trailingComma`) then I'm fine with that also.
```javascript
module.exports = {
  semi: true,
  trailingComma: "es5",
  singleQuote: true,
  printWidth: 120,
  tabWidth: 2
}
```